src/lang/en-US.ts

```diff
@@ -128,6 +128,7 @@ const LoginPage: ResolvedIntlConfig['messages'] = {
 
 const CaptureCreate: ResolvedIntlConfig['messages'] = {
     'captureCreation.ctas.discover': `Test Config`,
+    'captureCreation.ctas.discoverAgain': `Regenerate Catalog`,
```

Comment:
Would `'captureCreation.ctas.rediscover'` work here?

Reply:
I thought about that but wanted the key to feel more like it is a transformation on the key above and not a stand alone one... not sure if that even makes any sense though.